.github/workflows/push.yml

```diff
@@ -55,10 +55,11 @@ jobs:
 
       - name: Generate Documentation of Markdown pages
         run: |
-          yarn global add typedoc
-          yarn add typedoc-plugin-markdown
-          yarn typedoc --entryPoints src/components src/screens --out talawa-admin-docs --plugin typedoc-plugin-markdown --theme markdown --entryPointStrategy expand --exclude "**/*.test.ts" --exclude "**/*.css"
-          
+          npm install --global typedoc
+          npm install typedoc-plugin-markdown
+          npm install --save-dev @types/node
+          npx typedoc --entryPoints src/components src/screens --out talawa-admin-docs --plugin typedoc-plugin-markdown --theme markdown --entryPointStrategy expand --exclude "**/*.test.ts" --exclude "**/*.css"
```

Comment:
What have you changed to ensure that the typedoc generated files will always be MPX compatible during the giving action workflow process?

Reply:
Typedoc and its markdown plugin don't natively support MPX formatting. To address this, I am considering adding a post-processing script to escape `<`, `>`, `{`, and `}` characters in the generated Markdown files. This will be complemented by a validation step using npx docusaurus-mdx-checker to ensure MPX compatibility.

Reply:
1. Write a python script to do this
2. Follow the Google Python style guide especially in the DocString area. https://google.github.io/styleguide/pyguide.html
3. Make sure that the script is python `black`, `pydocstring`, `pylint`, `flake8` and `pycodestyle` compliant
4. Make sure that the script uses functions. Use the `countline.py` script in the `workflow` directory as a guide

![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/16875803/eee20105-1c68-467d-b1cb-9371240a0678)